spark/src/test/scala/geotrellis/spark/summary/polygonal/HistogramSpec.scala

```diff
@@ -72,44 +75,62 @@ class HistogramSpec extends FunSpec with TestEnvironment with TestFiles {
     }
 
     it("should get correct histogram over whole raster extent") {
-      val histogram = modHundred.polygonalHistogram(totalExtent.toPolygon)
+      // We use FastMapHistogram for this test because StreamingHistogram can shuffle bucket
```

Comment:
Theoretically we should swap _all_ the tests to use FastMapHistogramVisitor for the same reason. But then we don't have a way to validate that StreamingHistogramVisitor is generally correct. In any case, we've conflated the general correctness of the polygonal summary operation on RDDs with the specifics of each visitor implementation. This isn't a new problem, so it probably makes sense to improve later as the tests are sufficient for now.